tests/gui/test_widgets.py

```diff
@@ -372,18 +372,19 @@ def test_Loginbutton_on_clicked(mocker):
     lb.window.show_login.assert_called_once_with()
 
 
-def test_MainView_init():
+def test_MainView_init(mocker):
     """
     Ensure the MainView instance is correctly set up.
     """
-    mock_session_maker = mocker.MagicMock()
-    mv = MainView(None)
+    session_maker = mocker.MagicMock()
+    mv = MainView(session_maker, None)
```

Comment:
We have a `session_maker` fixture that I think we should use. I know we don't actually access it ever, but I don't think we should mock things unless we *really need to*. We've been bit by this before (tests passing because of mocks, but exceptions when the app is run).

Note that this is done in several places in this commit.

Reply:
Resolving now that this is outdated after removing session code from gui.